react-app/src/components/ActionIcon/index.module.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  width: 25px;
+  height: 25px;
+  background: url('./Apply.svg') center center/contain no-repeat;
+}
+
+$icons: (
+  apply: './Apply.svg',
```

Comment:
In order to better organize this component, I'd move these icons into an `images` subfolder.
```
- ActionIcon
  - index.js
  - index.module.scss
  - stories.js
  - images
    - Apply.svg
    - Back.svg
    ...
```

Reply:
Great tip! I will reorganize this as you suggest.